### PR TITLE
Replace "UpdateZone" by "UpdateZoneOptions"

### PR DESCRIPTION
**Pull request by Slewog**
I removed the function "UpdateZone" because she brokes the box zone and replaced it with "UpdateZoneOptions" which only allows updating targeting options (add, remove or update an option)

Here is a small example of how I use it on my server

image

### CODE REVIEW

File: client.lua [hunk 1]
```diff
@@ -897,7 +897,12 @@ exports("GetPlayer", function(label) return Players[label] end)
 
 exports("UpdateType", function(type, label, data) Types[type][label] = data end)
 
-exports("UpdateZone", function(name, data) Zones[name] = data end)
+local function UpdateZoneOptions (name, targetoptions)
+	targetoptions.distance = targetoptions.distance or Config.MaxDistance
+	Zones[name].targetoptions = targetoptions
+end
+
+exports("UpdateZoneOptions", function(name, targetoptions) UpdateZoneOptions(name, targetoptions) end)
```
Comment (BerkieBb) [inline client.lua:905]:
```suggestion
exports("UpdateZoneOptions", UpdateZoneOptions)
```
This is more minimalistic written, works the exact same as before

**Review by BerkieBb — APPROVED** [review]

File: client.lua (hunk 1 above)
Comment (Slewog) [reply client.lua:905]: Yep u update just the targetting options and not the complete zone ;)

**Comment by BerkieBb** [comment]
Aside from the change request I made, could you move this PR to target the develop branch instead?

**Comment by Slewog** [comment]
> Aside from the change request I made, could you move this PR to target the develop branch instead?

How i can do it ? I'm not familiar with github commands and I'm making the changes from the website

**Comment by BerkieBb** [comment]
I managed to change it, you do it by pressing `Edit` at the top next to the title and then change the branch under the title of the PR

**Comment by Slewog** [comment]
> 

Ok thank's for the tips

**Pull request merged**